### ISSUE TITLE
Add migrate script to pre-deploy

### ISSUE THREAD

**Issue by DavidTheProgrammer** [CodeForAfrica/wazimap-ng#13]
Add a migrate command to the `predeploy` script to ensure any pending migrations are always run at startup.

**Comment by satoru260** [comment]
@kilemensi, @thepsalmist thoughts on this? The deployment had some hiccups because the application didn't run migrations at startup. There's already a predeploy script that runs `collectstatic`, I'm considering adding  a `migrate` command to it as wel..

**Comment by kilemensi** [comment]
My 2 cents would be to keep this as issue for now. We'll pick it up when we discuss the future of the project i.e. lets not make any changes not already in/incompatible with upstream.

I believe migrations can be run manually after you've ssh'ed into the instance/app, right @DavidTheProgrammer ?

**Comment by satoru260** [comment]
That's fair, I think we can leave it as an issue for now as advised. We can circle back to it if needed.

Yes, they can. That's how i resolved it this time around. I'll have to remember to do it again when we switch to the final DB

**Comment by thepsalmist** [comment]
@DavidTheProgrammer 👍  I see they have a migrate command within the [app.json](https://github.com/OpenUpSA/wazimap-ng/blob/staging/app.json) but i'll double also check

**Comment by satoru260** [comment]
> [@DavidTheProgrammer](https://github.com/DavidTheProgrammer) 👍 I see they have a migrate command within the [app.json](https://github.com/OpenUpSA/wazimap-ng/blob/staging/app.json) but i'll double also check

Yes, but it's under the "review" section, so I believe it only applies to review apps.
